src/main/java/com/service/PersonService.java

```diff
@@ -93,4 +97,24 @@ public void showPersons() {
             System.out.println(person.toString());
         }
     }
+
+    @Override
+    public void showPersonsFromFile() {
+
+        String fileLoc = "C:\\Петко\\udemy\\java master class\\Martin_Project\\personsList.txt";
```

Comment:
Create resource foulder and in it create **application.properties** file. Research it and check how to read properties from there. Move this fileLoc string there and name it more appropriately

Reply:
Actually do this for every file that you use